dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/UnpackagedTests/TestClass.cs

```diff
@@ -107,47 +107,89 @@ public class TestClass
     {
         private ActivationContext m_context = new ActivationContext();
         private static string m_assemblyFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        private static string m_exeFolder = Path.GetDirectoryName(System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName);
+        private static bool m_rs5 = false;
```

Comment:
nit: maybe something like `m_rs5_or_lower` since it checks for "< 18362" ?

Reply:
anything below rs5 doesn't exist for WindowsAppSDK :)